docs/release-notes.md

```diff
@@ -9,20 +9,54 @@ description: >
 We recommend always using "tagged" versions of Ondat rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-The latest tagged release is `2.8.0`. For
+The latest tagged release is `2.8.1`. For
 installation instructions see our
 [Install](/docs/reference/cluster-operator/install) page.
 
-The latest CLI release is `2.8.0`, available from
+The latest CLI release is `2.8.1`, available from
 [GitHub](https://github.com/storageos/go-cli/releases).
 
 # Upgrading
 
 To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.
 
-## 2.8.0 - Release 2022-06-29
+## 2.8.1 - Release 2022-08-02
+
+### New
+
+k8s
+
+* Install resource quota for GKE
+* Disable scheduler option in storageoscluster CR
+* Operator:
+  * Add snapshot controller and related CRDs if not present
+  * Pod Disruption Budget support for k8s v1.25
+* Improve logging on kubectl plugin
+
+Data Plane
+
+* Warn that filesystem might go read-only after a failed write/unmap/sync SCSI command. The log of interest is: `"SCSI command failed - if the block device is mounted the filesystem may go read-only".`
```

Comment:
"unmap" needs to be removed so the sentence reads: "Warn that filesystem might go read-only after a failed write or sync SCSI command"

Reply:
updated